devices/linux/Files/main.py

```diff
@@ -1,42 +1,47 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-import subprocess 
+
+import base64
+import getpass
+import os
+import platform
+import re
+import subprocess
 import sys
+import uuid
+from shutil import copyfile
 
-def missing_dbus():
-    print("Cannot import the dbus module")
-    sys.exit(1)
-    
 try:
     import dbus
-except:
+except ImportError:
     if sys.version_info.major == 3:
-        missing_dbus()
+        print("Cannot import the dbus module")
+        sys.exit(1)
     try:
         subprocess.call(['python3'] + sys.argv)
     except:
-        missing_dbus()
+        print("Cannot import the dbus module")
```

Comment:
I know this is as it was before but it stays the wrong message :). Also: I _think_ this should be shown to the user (which might have called us from nautilus or so) in a GUI.